### ISSUE TITLE
More then 1 account in script

### ISSUE THREAD

**Issue by aboldin** [AHAAAAAAA/PokemonGo-Map#1159]
I don't see open issues about this question, but how can we improve this script to launch it with 2 or more account and coordinates used?
At this moment a have 26 accounts scanning my city, but they are all on different ports.. Not very comfortable

**Comment by Whoknowsit** [comment]
Multiple accounts are not required, but scanning multiple locations using one process would be nice.

**Comment by aboldin** [comment]
> Multiple accounts are not required, but scanning multiple locations using one process would be nice.

Using just one account is not good idea -  it will be banned very fast if it flies all over the city with 500kmh speed )

**Comment by uzanief** [comment]
> At this moment a have 26 accounts scanning my city, but they are all on different ports.. Not very comfortable

yeah i was doing the same thing... it feels like some kind of secret agent activity, pretty weird... but I still believe that multiple accounts are neccessary.

**Comment by kevvv** [comment]
Not to mention multiple accounts would speed up the scans drastically.

**Comment by welshboy** [comment]
So multiple accounts, scanning and plotting on a single map?

**Comment by Whoknowsit** [comment]
> Using just one account is not good idea - it will be banned very fast if it flies all over the city with 500kmh speed )

Huh? Reading the data from their API doesn't trigger any "movement".

**Comment by welshboy** [comment]
@Whoknowsit In order to get the pokemon in a specific area, the programs sends co-ordinates to pogo servers, which in return gives info about the surrounding area. In order to scan multiple areas, you have to send multiple co-ordinates to pogo servers. This will show up as movement, and as the person says, if you move 1m in a second then they know something is up

**Comment by Valareos** [comment]
You do not need to run them all on different ports. In fact, if you run say 5 different accounts, looking at 5 different locations, but all ran from the same working directory with same port, it will combine that info, as they end up using the same database. In other words, multiple accounts, multiple processes, 1 map, 1 port

**Comment by slonce** [comment]
And my question is this - how to implement so that every person who comes to the map, could separately for themselves to put the location, or at least hide Pokemon only for themselves but not for everyone. Because when hiding Pokémon on the map it hides Pokemon fully to all who sign up on the map. That is, the code rules for all. And you can do that everyone could separately configure for themselves? Thank you

**Comment by captcho** [comment]
@Valareos there is already implemented solution to login through multiple accounts?

**Comment by aboldin** [comment]
Yeap, that works
http://188.225.76.41:8000/

**Comment by captcho** [comment]
@aboldin how did you do this?

**Comment by AnakTeka** [comment]
@aboldin Is this running on same port and working directory? 
It keeps saying 
`socket.error: [Errno 98] Address already in use` whenever I try to run another instance running on the same port.

**Comment by aboldin** [comment]
@captcho @AnakTeka Just use different port for launch =)
I made it like this
superscript.sh:

> nohup python runserver.py -a google -u *** -p *** -l "pr. Karla Marksa, 1, Novosibirsk, Novosibirskaya oblast', 630064" -st 12 -P 8000 &
> sleep 3
> nohup python runserver.py -a google -u *** -p *** -l "ul. Dusi Kovalchuk, 179/4, Novosibirsk, Novosibirsk Oblast" -st 12 -P 8001 &
> sleep 3
> nohup python runserver.py -a google -u *** -p *** -l "ul. Vokzalnaya magistral', 1, Novosibirsk, Novosibirsk Oblast" -st 12 -P 8002 &
> sleep 3
> .........

**Comment by danielbennifer** [comment]
@aboldin How would one make use of this script? Interested in checking this out. Would be far more preferable, than having to continue changing location each time.
